spec/features/sessions/user_logs_out_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.feature do
+  context 'user' do
+    scenario 'logs out' do
+      user = create(:user)
+
+      visit login_path
+
+      fill_in :email, with: user.email
+      fill_in :password, with: 'password'
+      click_on 'Enter'
+
+      click_on 'Logout'
+
+      expect(current_path).to eq root_path
+      expect(page).to have_link 'Login'
+      expect(page).to_not have_content "Logged in as #{user.first_name}"
```

Comment:
I like to check non expectations with expectations in the beginning, e.g., `expect(page).to have_content "logged in as..."` on line 13 or something